scripts/combine/setupCombine.py

```diff
@@ -41,7 +41,8 @@ def make_parser(parser=None):
     parser.add_argument("--fitresult", type=str, default=None ,help="Use data and covariance matrix from fitresult (for making a theory fit)")
     parser.add_argument("--noMCStat", action='store_true', help="Do not include MC stat uncertainty in covariance for theory fit (only when using --fitresult)")
     parser.add_argument("--fakerateAxes", nargs="+", help="Axes for the fakerate binning", default=["eta","pt","charge"])
-    parser.add_argument("--ABCD", action="store_true", help="Produce datacard for simultaneous fit of ABCD regions")
+    parser.add_argument("--simpleABCD", action='store_true', help="Do the simple ABCD method, default is extendedABCD")
```

Comment:
Since we start having multiple options, what about having a single one with multiple choice ?

Reply:
The two options are orthogonal and could in principle be used both at the same time. The simultaneousABCD means the separate regions are fit simultaneous and we have explicit nuisance parameters for the fake estimation. The simple or extended ABCD is a switch.